src/components/dashboard/main-content/course-enrollments/course-cards/unarchive-modal/ModalBody.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+
+import ModalError from './ModalError';
+import UnarchiveModalContext from './UnarchiveModalContext';
+
+const ModalBody = () => {
+  const {
+    confirmError,
+    courseLink,
+    courseTitle,
+  } = useContext(UnarchiveModalContext);
+  return (
+    <>
+      {confirmError && <ModalError />}
+      <p className="m-0">
+        Are you sure you want to unarchive
+        {' '}
+        <a href={courseLink}>{courseTitle}</a>?
+        {' '}
+        The course will show up in your &quot;In Progress&quot; section.
+      </p>
+      <p className="mt-1">
+        As long as
```

Comment:
nit: similar as other comment, can this message be reduced to a fewer number of lines?